templates/webapi/test/infopanel.html.ep

```diff
@@ -47,11 +47,12 @@
                         % my $state_detail = '';
                         % if (my @gru_deps = $job->gru_dependencies) {
                             % $state_detail = join(';', map {
-                            %   'id: ' . $_->id . ', name: ' . $_->taskname
+                            %   my $link = url_for('/minion/jobs')->query(note => 'gru_id_' . $_->id, task => $_->taskname);
```

Comment:
Now I'm wondering - not everyone can access the minion dashboard. Maybe the link could only be printed `if (is_operator)`. Presenting a link that one cannot access is maybe not so nice.